modules/ROOT/pages/provisioning-virtualbox.adoc

```diff
@@ -4,193 +4,57 @@ This guide shows how to provision new Fedora CoreOS (FCOS) nodes on the VirtualB
 [source, bash]
 ----
-sudo vi "${TMP_FCOS_BOOT_VOLUME}/loader/entries/ostree-1-fedora-coreos.conf"
-# ...
-sudo umount "${TMP_FCOS_BOOT_VOLUME}"
-----
-====
+IGN_PATH="SetWithABSPathToIGNConfig"
+VM_NAME="Fedora CoreOS"
 
-Continue with converting the raw disk image into a VirtualBox disk image:
-[source, bash]
+VBoxManage guestproperty set ${VM_NAME} /Ignition/Config "$(cat ${IGN_PATH}})"
 ----
-vboxmanage convertfromraw "${UNPACKED_DISK}" "${VM_DISK}" --format VDI --variant Standard
-# Resize the disk to match FCOS requirements
-vboxmanage modifymedium disk "${VM_DISK}" --resize 10240
-# Add the converted disk to the VM
-vboxmanage storageattach "${VM_NAME}" --storagectl SATA --type hdd --port 0 --device 0 --medium "${VM_DISK}"
-----
-
-TIP: You can use https://www.virtualbox.org/manual/UserManual.html#vboxmanage-snapshot[snapshots] and https://www.virtualbox.org/manual/UserManual.html#vboxmanage-clonevm[cloning] in case you want to reuse the VM as an empty baseline before applying an Ignition configuration to it.
-
 [NOTE]
 ====
-*Fedora CoreOS requires the root filesystem to be at least 8 GiB.* For practical reasons, disk images for some platforms ship with a smaller root filesystem, which by default automatically expands to fill all available disk space. If you add additional partitions after the root filesystem, you must make sure to explicitly resize the root partition as shown so that it is at least 8 GiB.
-
-Currently, if the root filesystem is smaller than 8 GiB, a warning is emitted on login. Starting from June 2021, if the root filesystem is smaller than 8 GiB and is followed by another partition, Fedora CoreOS will refuse to boot. For more details, see https://github.com/coreos/fedora-coreos-tracker/issues/586[this bug].
+Fedora CoreOS is using the guest properties to import the Ignition config. There are character limits to consider. 
+If your Ignition configuration is to large then you will need to follow the appropriate steps for doing a config merge. 
+You can read more about this process here https://docs.fedoraproject.org/en-US/fedora-coreos/remote-ign/
 ====
 
-You can now boot the target system you previously configured.
 
-If you decided to supply the Ignition configuration URL dynamically or want to override your static URL, press `e` when the GRUB menu shows up. This allows you to edit the current boot entry and specify the remote Ignition configuration URL to use:
-[source, bash]
-----
-load_video
-#...
-linux ($root)/ostree/fedora-coros- #...
- ignition.firstboot ignition.config.url=http://10.0.2.2:8000/example.ign
-initrd ($root)/ostree/fedora-coreos- #...
-----
-
-The actual first boot process begins now. It is at this time that Ignition ingests the configuration file and provisions the system as specified.
-
-Once the VM finished booting, you can log in from your host:
-[source, bash]
-----
-ssh core@localhost -p 2222
```

Comment:
If we keep the NAT port forwarding case, we should probably keep this too as a hint.